saleor/core/utils/__init__.py

```diff
@@ -136,6 +135,8 @@ def serialize_decimal(obj):
 
 
 def create_superuser(credentials):
+    from ...account.models import User
```

Comment:
Is it expected?

Reply:
Yes, otherwise their would be a circular import. Utils files in saleor are always a cause of circular imports as they contain mostly core/heavy/important logic, we should look forward to refactoring them into proper files (utils files are often a bad practice).